src/Language/PureScript/Errors.hs

```diff
@@ -636,6 +638,15 @@ prettyPrintSingleError (PPEOptions codeColor full level showWiki) e = flip evalS
         where
         go TUnknown{} = True
         go _ = False
+    renderSimpleErrorMessage (AmbiguousTypeVariables ident t (Constraint nm ts _)) =
+      paras [ line $ markCode (showIdent ident) ++ " doesn't have a type declaration, so it was inferred to be of type: "
```

Comment:
I would omit this line, since it will just duplicate the information from the value declaration level (` ... in declaration x ...`) anyway. In fact, I'd remove the `Ident` from the `SimpleErrorMessage` constructor.